app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -124,16 +140,43 @@ def organization
   end
   helper_method :organization
 
+  def classroom_config
+    starter_code_repo_id = group_assignment_repo.starter_code_repo_id
+
+    return nil unless starter_code_repo_id
+
+    client       = group_assignment_repo.creator.github_client
+    starter_repo = GitHubRepository.new(client, starter_code_repo_id)
+
+    ClassroomConfig.new(starter_repo)
```

Comment:
Can we memoize this? Right now we only call this method once, but in the future if we add another reference, we wouldn't want all these API calls to be made again.

`@classroom_config ||= ClassroomConfig.new(starter_repo)`